.github/workflows/nodeCI.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        node-version: [14.x]
+        node-version: [14.x, 16.x]
```

Comment:
can you remove 14.x? otherwise we will continue to check it against 14 in the CI

Reply:
Sure

Reply:
Done